src/main/kotlin/com.bugsnag.android.gradle/BugsnagManifestUuidTask.kt

```diff
@@ -59,10 +60,10 @@ open class BugsnagManifestUuidTask @Inject constructor(objects: ObjectFactory) :
     fun updateManifest() {
         val manifestPath = getManifestPaths(project, variant, variantOutput)
         if (manifestPath == null) {
-            project.logger.warn("Bugsnag: Failed to find manifest at $manifestPath for $variantOutput")
+            logger.warn("Bugsnag: Failed to find manifest at $manifestPath for $variantOutput")
```

Comment:
While this task isn't compatible with configuration caching (not an issue since it wouldn't be used on versions that support it anyway), I did clean this up just to make it more consistent with other tasks